tests/test_deferredCharge.py

```diff
@@ -98,6 +98,20 @@ def setUp(self):
         self.calib.decayTime['amp1'] = 3.2
         self.calib.globalCti['amp0'] = 1.4e-7
         self.calib.globalCti['amp1'] = 1.5e-7
+        self.calib.signals['amp0'] = np.linspace(1.0e2, 1.0e5, 10)
+        self.calib.signals['amp1'] = np.linspace(1.0e2, 1.0e5, 10)
+        self.calib.serialEper['amp0'] = np.full_like(self.calib.signals['amp0'], 2.0e-6)
+        self.calib.serialEper['amp1'] = np.full_like(self.calib.signals['amp1'], 2.0e-6)
+        self.calib.parallelEper['amp0'] = np.full_like(self.calib.signals['amp0'], 1.0e-6)
+        self.calib.parallelEper['amp1'] = np.full_like(self.calib.signals['amp1'], 1.0e-6)
+        self.calib.serialCtiTurnoff['amp0'] = 1.0e5
+        self.calib.serialCtiTurnoff['amp1'] = 1.0e5
+        self.calib.parallelCtiTurnoff['amp0'] = 1.0e5
+        self.calib.parallelCtiTurnoff['amp1'] = 1.0e5
+        self.calib.serialCtiTurnoffSamplingErr['amp0'] = 1.0e3
+        self.calib.serialCtiTurnoffSamplingErr['amp1'] = 1.0e3
+        self.calib.parallelCtiTurnoffSamplingErr['amp0'] = 1.0e3
+        self.calib.parallelCtiTurnoffSamplingErr['amp1'] = 1.0e3
```

Comment:
After reading through the rest, I think this probably should be fixed here (so `amp1` gets a `logistic` trap).  Adding a `testPartiallyPopulated` might be good to add to test that `serialTraps` can be not populated for `amp1` to keep the accidental test case intact.